OpenRA.Mods.Common/Traits/Attack/AttackFollow.cs

```diff
@@ -27,6 +27,9 @@ public class AttackFollowInfo : AttackBaseInfo
 		[Desc("Keep firing on targets even after attack order is cancelled")]
 		public readonly bool PersistentTargeting = true;
 
+		[Desc("Range to stay away from min and max ranges to give some leeway if the target starts moving.")]
+		public readonly WDist LeewayRange = new WDist(1024);
```

Comment:
The name `LeewayRange` doesn't sound right to me, but I don't have an immediately better suggestion. Maybe something with "Inset" in it?

Also `WDist.FromCells(1)`?

Reply:
Perhaps `LeewayMargin`

Reply:
"Leeway" was specifically the part that I didn't like here!

Reply:
How about `RangeMargin` instead?

Reply:
I don't have a strong opinion either way.

Reply:
Updated.